src/vm/util.cpp

```diff
@@ -1877,7 +1877,7 @@ size_t GetCacheSizePerLogicalCpu(BOOL bTrueSize)
         }
     }
 
-#if defined(_TARGET_AMD64_) || defined (_TARGET_X86_)
+#if defined (_TARGET_X86_)
     DefaultCatchFilterParam param;
```

Comment:
If you are removing `_AMD64_` here, you can also remove `#ifdef _WIN64` inside this block.

Reply:
right...I haven't because this is for preview and there is a chance we might still want to do this after our perf testing. if the perf testing shows that there's no need to do this, I'll do some code clean up then.